datastore/google/cloud/datastore/transaction.py

```diff
@@ -234,3 +282,36 @@ def commit(self):
         finally:
             # Clear our own ID in case this gets accidentally reused.
             self._id = None
+
+    def put(self, entity):
+        """Remember an entity's state to be saved during :meth:`commit`.
```

Comment:
Rather than copy-paste it'd be better to just say this is `Batch.put` PLUS some extra features.

Reply:
ok.